modules/KalturaSupport/components/dualScreen/imagePlayer.js

```diff
@@ -59,8 +59,11 @@
 						return false;
 					}
 				});
+
 				if (!cuePoint){
-					cuePoint = _this.getCurrentCuePoint();
+					// TODO [es]: TBD with Oren - I removed this since reached event will also trigger for code cue points and if I understood you correctly this workaround is redundant
+					// cuePoint = _this.getCurrentCuePoint();
```

Comment:
@OrenMe I want to make sure that this workaround is not needed anymore since this function will be invoked also for the new code cue points and I don't want it to start searching for thumbs each time the new cue points is reached